src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -202,18 +202,36 @@ internal SyntaxNode MethodDeclaration(IMethodSymbol method, string name, IEnumer
             throw new NotImplementedException();
         }
 
+        private protected abstract SyntaxNode OperatorDeclaration(
+            int syntaxKind,
```

Comment:
i think i'd prefer just passing hte operator name along.  thsi also includes 'checked' so it can subsume the first two parameters.